docs/guide/installation.rst

```diff
@@ -0,0 +1,103 @@
+.. currentmodule:: discord
+
+.. _guide_install:
+
+Installation
+=============
+
+Welcome to discord.py, a library for Python to aid in creating applications for Discord.
+
+Prerequisites
+--------------
+
+To install discord.py you'll need Python version 3.8 or higher. Earlier versions of Python are not supported.
+
+.. _guide_install_primer:
+
+Primer
+-------------
+
+On Unix systems you can run the following command to install discord.py from PyPI.
+
+.. code-block:: shell
+
+    python3 -m pip install -U discord.py
+
+On Windows systems, you can use the following command instead.
+
+.. code-block:: shell
+
+    py -3 -m pip install -U discord.py
+
+Voice Support
+~~~~~~~~~~~~~~
+
+Voice support (e.g. playing audio in voice channels) is not enabled by default and can be enabled by installing ``discord.py[voice]`` instead of ``discord.py``. ::
+
+    pip install -U "discord.py[voice]"
+
+Linux systems may need to install additional dependencies via your package manager to get full voice support:-
+
+.. tab:: Debian/Ubuntu
+
+    .. code-block:: shell
+
+        $ sudo apt install libffi-dev libsodium-dev python3-dev
+
+.. tab:: Fedora/CentOS
+
+    .. code-block:: shell
+
+        $ sudo dnf install libffi-devel libsodium-devel python3-devel
+
+.. tab:: Arch Linux
+
+    .. code-block:: shell
+
+        $ pacman -Syu libsodium
```

Comment:
Is this a necessary addition? If so, what's the name of the package to add?
(I do not use Arch nor can I test)

Reply:
If `libffi` is marked as required for all of the other systems, I would assume that it would be the same case for Arch. The name of the package is [`libffi`](<https://archlinux.org/packages/core/x86_64/libffi/>)